frontend/src/page/Home.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {useEffect, useState} from 'react';
+
+import {Base} from "../component/Base";
+import {BotInfo, useApi} from "../service/useApi";
+import {BotInfoGrid} from "../component/botInfo/BotInfoGrid";
+
+export default function Home() {
+    const {fetchBotInfo} = useApi();
+
+    const [stats, setStats] = useState<BotInfo|null>();
+
+    useEffect(() => {
+        fetchBotInfo().then(result => {
+            setStats(result);
+        });
+    }, []);
+
+    return (
+        <Base>
+            {stats != null ? <BotInfoGrid botInfoStats={stats!} /> : <div>Loading...</div>}
+        </Base>
+    );
+}
```

Comment:
```suggestion
export default function Home() {
    const {fetchBotInfo} = useApi();

    const stats = use(fetchBotInfo());

    return (
        <Base>
            <Suspense fallback={<div>Loading...</div>}>
              <BotInfoGrid botInfoStats={stats} />}
            </Suspense>
        </Base>
    );
}
```

Reply:
async/await is not yet supported in Client Components, only Server Components. This error is often caused by accidentally adding `'use client'` to a module that was originally written for the server.